docs/whirlpool/src/theme/Root.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from "react";
+
+export default function Root({ children }) {
+  const [isHovered, setIsHovered] = useState(false);
+
+  const buttonStyle = {
+    padding: "10px 20px",
+    backgroundColor: "#FFD15C",
+    color: "#13174D",
+    fontSize: "18px",
+    fontWeight: "500",
+    border: "1px solid #13174D",
+    borderRadius: "8px",
+    cursor: "pointer",
+    boxShadow: "3px 2px 6px rgba(0, 0, 0, 0.3)",
+    transition: "transform 0.2s ease, box-shadow 0.2s ease",
+    ...(isHovered && {
+      transform: "translate(2px, 1px)",
+      boxShadow: "None",
+    }),
+  };
+
+  useEffect(() => {
+    const script = document.createElement("script");
+    script.src = "https://tally.so/widgets/embed.js";
```

Comment:
Might be safer to just use a next `<Script>` tag (https://nextjs.org/docs/app/api-reference/components/script)